### DISCUSSION TITLE
Leveraging Mitosis with Expo SDK 48+: Seeking Tips for Design System Integration

### DISCUSSION THREAD

**Discussion by jfbaro** [BuilderIO/jsx-lite#1215] [General]
Hey there,

Is anyone currently using Mitosis with Expo SDK 48+? We've adopted Expo SDK as our go-to framework for new projects encompassing both Web and Mobile development. However, we still maintain legacy systems using Angular and React. Now, I'm looking to standardize our design system using Mitosis, ensuring it can be seamlessly utilized by other teams working with Angular or React. From now on, our primary focus is Expo (Web and Mobile).

If you have any tips or advice on integrating Mitosis into Expo SDK 48+ projects, I'd greatly appreciate your insights.

Thanks in advance!

**Comment by samijaber** [comment]
I don't think there's anything specific to Expo to keep in mind, besides setting it up correctly. Here's an Expo integration test that integrates a Mitosis-generated React Native Builder SDK, feel free to browse for inspiration:

https://github.com/BuilderIO/builder/tree/f4804b0ddd8438d9d3582f0ebbd23de84159a2f6/packages/sdks/e2e/react-native
